internal/logger/audit.go

```diff
@@ -145,7 +144,7 @@ func AuditLog(ctx context.Context, w http.ResponseWriter, r *http.Request, reqCl
 	// Send audit logs only to http targets.
 	for _, t := range auditTgts {
 		if err := t.Send(ctx, entry); err != nil {
-			LogAlwaysIf(context.Background(), fmt.Errorf("event(%v) was not sent to Audit target (%v): %v", entry, t, err), madmin.LogKindAll)
+			LogOnceIf(ctx, fmt.Errorf("Unable to send an audit event to the target `%v`: %v", t, err), "send-audit-event-failure")
```

Comment:
We are logging missed events such that they are not lost, if we do LogOnceIf() then we lose such logs.

Reply:
> We are logging missed events such that they are not lost, if we do LogOnceIf() then we lose such logs.

I understand, but audit is really verbose, these are a lot of events. This will also flood the http logger. I am not sure if we should care about printing this audit in the log.

Reply:
yeah its a trade-off we don't know if we should preserve or lose these events.